msos/MixedStack.cs

```diff
@@ -57,20 +59,24 @@ public static void VerifyHr(int hr)
     class UnifiedStackFrame
     {
         public UnifiedStackFrameType Type { get; set; }
-        
+
         public string Module { get; set; }
         public string Method { get; set; }
 
         public ulong OffsetInMethod { get; set; }
         public ulong InstructionPointer { get; set; }
         public ulong StackPointer { get; set; }
 
+        public ulong FrameOffset { get; private set; }
         public string SourceFileName { get; set; }
         public uint SourceLineNumber { get; set; }
         public uint SourceLineNumberEnd { get; set; }
         public uint SourceColumnNumber { get; set; }
         public uint SourceColumnNumberEnd { get; set; }
 
+        public List<byte[]> NativeParams { get; set; }
+        public List<UnifiedHandle> Handles { get; set; }
```

Comment:
It doesn't feel right that these parameters are hanging off `UnifiedStackFrame`. Not all frames have handles, and the `NativeParams` thing works only for x86 anyway. Can you find some better place for this stuff? Perhaps closer to where it is being used?